<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObjViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="pic">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>39</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="trace">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>tracing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simplify">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>simplifying</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>670</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>560</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_up">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>shift up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_left">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>shift left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_right">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>shift right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_down">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>shift down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_left">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_right">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orbit_left">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>orbit left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orbit_right">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>orbit right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twist_left">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>twist clkwise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twist_right">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>twist counterclkwise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_in">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>zoom in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_out">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>zoom out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="open_action">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="quit_action">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
